src/test/java/BouquetCreatorTest.java

```diff
@@ -0,0 +1,46 @@
+import model.Bouquet;
+import model.DecorationPlant;
+import model.Flower;
+import model.FlowerOne;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static model.FlowerColor.RED;
+import static model.FlowerColor.WHITE;
+import static org.junit.Assert.assertTrue;
+
+public class BouquetCreatorTest {
+    BouquetCreator bouquetCreator;
+    List<Flower> availableFlowers;
+    List<DecorationPlant> availableDecorations;
+    Bouquet outputBouquet;
+
+
+    @Before
+    public void prepare() {
+        FlowerCreator flowerCreator = new FlowerCreator();
```

Comment:
you can't test many classes in one test, and there are should be asserts, otherwise test doesn't make sense (will never fail)